test/ffmpeg-qsv/encode/vp9.py

```diff
@@ -125,3 +125,38 @@ def test(self, case, gop, bitrate, fps, slices, refmode, quality, looplvl, loops
     self.init(spec, case, gop, bitrate, fps, slices, quality)
     self.encode()
 
+class seek(VP9_8EncoderTest):
+  def init(self, tspec, case, ipmode, qp, quality, slices, seek):
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      case      = case,
+      gop       = 30 if ipmode != 0 else 1,
+      qp        = qp,
+      slices    = slices,
+      rcmode    = "cqp",
+      quality   = quality,
+      seek      = seek,
+    )
+
+  @parametrize_with_unused(*gen_vp9_seek_parameters(spec), ['refmode', 'loopshp', 'looplvl'])
+  def test(self, case, ipmode, qp, quality, slices, refmode, looplvl, loopshp, seek):
```

Comment:
For seek tests, let's just remove `refmode, loopshp, and looplvl` from the test parameters.